docling_core/types/doc/document.py

```diff
@@ -677,6 +678,89 @@ class PictureItem(FloatingItem):
 
     annotations: List[PictureDataType] = []
 
+    # Convert the image to Base64
+    def _image_to_base64(self, pil_image, format="PNG"):
+        buffered = BytesIO()
+        pil_image.save(buffered, format=format)  # Save the image to the byte stream
+        img_bytes = buffered.getvalue()  # Get the byte data
+        img_base64 = base64.b64encode(img_bytes).decode(
+            "utf-8"
+        )  # Encode to Base64 and decode to string
+        return img_base64
+
+    def export_to_markdown(
+        self,
+        doc: "DoclingDocument",
+        add_caption: bool = True,
+        image_placeholder: str = "<!-- image -->",
+        image_mode: ImageRefMode = ImageRefMode.EMBEDDED,
+        filename: Optional[Path] = None,
+    ):
+        r"""Export picture to Markdown format."""
+        if image_mode == ImageRefMode.PLACEHOLDER:
+            text = "\n" + image_placeholder + "\n"
+            return text
+
+        elif image_mode == ImageRefMode.EMBEDDED and isinstance(self.image, ImageRef):
+            text = f"![Local Image]({self.image.uri})\n"
+            return text
+
+        elif image_mode == ImageRefMode.EXPORTED_TO_PNG and isinstance(
+            self.image, ImageRef
+        ):
+            img = self.get_image(doc)
+            if (img is not None) and (filename is not None):
+                img.save(filename)
+
+                text = f"![Local Image]({filename})\n"
+                return text
+            else:
+                text = f"<!-- can not use {filename} as image-filename --> "
+                return text
+
+        elif image_mode == ImageRefMode.EMBEDDED and not isinstance(
+            self.image, ImageRef
+        ):
+            text = (
+                "<!-- 🖼️❌ Image not available. "
+                "Please use `PdfPipelineOptions(generate_picture_images=True)`"
+                " --> "
+            )
+            return text
+
+    def export_to_html(
+        self,
+        doc: "DoclingDocument",
+        add_caption: bool = True,
+        filename: Optional[Path] = None,
```

Comment:
As with `export_to_markdown`, putting a filename argument here is not a proper solution.